Rakefile

```diff
@@ -53,6 +53,8 @@ def create_hpxmls
     'invalid_files/unattached-window.xml' => 'base.xml',
     'invalid_files/water-heater-location.xml' => 'base.xml',
     'invalid_files/water-heater-location-other.xml' => 'base.xml',
+    'invalid_files/desuperheater_1_speed_invalid_relatedHVAC.xml' => 'base-hvac-central-ac-only-1-speed.xml',
+    'invalid_files/desuperheater_1_speed_repeated_relatedHVAC.xml' => 'base-hvac-central-ac-only-1-speed.xml',
```

Comment:
Great to see these 👍. But can you use dashes instead of underscores for consistency with all the other files?